mantidproject/mantid
# Remove (unused) generic Sequential Fit button from fit menu in EngDiff UI

**Description of work.**

Small change to remove the generic Sequential Fit button in the fit menu - the users of EngDiff fitting should use the button in the UI that takes the runs from the UI table.


**To test:**

(1) Interfaces > Diffraction > Engineering Diffraction > Fitting
(2) Load in a focussed run (pick any in [ENGINX_305761_bank_1.zip](https://github.com/mantidproject/mantid/files/5764508/ENGINX_305761_bank_1.zip))
(3) Plot the run 
(4) Click Fit on the plot toolbar
(5) Click Fit in the browser that appears on the right
(6) Check Sequential Fit no longer an option


Fixes #30044


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
